### PR TITLE
fix(FEC-10994): remove uiconf specific for kaltura player data

### PR DESCRIPTION
**Pull request by Yuvalke**

### CODE REVIEW

**Review by yairans — APPROVED** [review]

**Review by OrenMe — APPROVED** [review]

File: alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php [hunk 1]
```diff
@@ -27,6 +27,7 @@ class embedPlaykitJsAction extends sfAction
 	const KALTURA_OVP_PLAYER = 'kaltura-ovp-player';
 	const KALTURA_TV_PLAYER = 'kaltura-tv-player';
 	const NO_ANALYTICS_PLAYER_VERSION = '0.56.0';
+	const NO_UICONF_FOR_KALTURA_DATA = '1.8.0';
```
Comment (yairans) [inline alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php:30]:
```suggestion
	const NO_UICONF_FOR_KALTURA_DATA = '1.9.0';
```

File: alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php [hunk 2]
```diff
@@ -173,8 +174,18 @@ private function appendConfig($content, $i18nContent)
 		$confNS = "window.__kalturaplayerdata";
 		$content .= "
 		$confNS = ($confNS || {});
-		$confNS.UIConf = ($confNS.UIConf||{});$confNS.UIConf[\"" . $this->uiconfId . "\"]=$uiConfJson;
 		";
+		$kalturaPlayerVersion = isset($this->bundleConfig[self::KALTURA_OVP_PLAYER]) ? $this->bundleConfig[self::KALTURA_OVP_PLAYER] : $this->bundleConfig[self::KALTURA_TV_PLAYER];
+		// For player latest/beta/canary or >= 1.8.0
```
Comment (yairans) [inline alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php:179]:
```suggestion
		// For player latest/beta/canary or >= 1.9.0
```

**Review by yairans — CHANGES_REQUESTED** [review]

File: alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php (hunk 2 above)
Comment (yairans) [reply alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php:179]: @Yuvalke change the base to Quasar-17.5.0

File: alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php [hunk 3]
```diff
@@ -173,8 +174,15 @@ private function appendConfig($content, $i18nContent)
 		$confNS = "window.__kalturaplayerdata";
 		$content .= "
 		$confNS = ($confNS || {});
-		$confNS.UIConf = ($confNS.UIConf||{});$confNS.UIConf[\"" . $this->uiconfId . "\"]=$uiConfJson;
 		";
+
+		$kalturaPlayerVersion = isset($this->bundleConfig[self::KALTURA_OVP_PLAYER]) ? $this->bundleConfig[self::KALTURA_OVP_PLAYER] : $this->bundleConfig[self::KALTURA_TV_PLAYER];
+		// For player latest/beta/canary or >= 1.9.0
```
Comment (yairans) [inline alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php:180]: remove comment

**Review by yairans — CHANGES_REQUESTED** [review]

**Review by yairans — APPROVED** [review]

**Pull request merged**